sympy/liealgebras/cartan_base.py

```diff
@@ -0,0 +1,108 @@
+from sympy.core import Basic
+from sympy.core.backend import zeros, Matrix
+
+class Standard_Cartan(Basic):
+    """
+    Semi-Concrete base class for Cartan types such as A4, etc
+    """
+
+    def __new__(cls, series, n):
+        obj = Basic.__new__(cls, series, n)
+        obj.n = n
+        obj.series = series
+        return obj
+
+    def rank(self):
+        """
+        Returns the rank of the Lie algebra
+        """
+        return self.n
+
+    def series(self):
+        """
+        Returns the type of the Lie algebra
+        """
+        return self.series
+
+    def cartan_matrix(self):
+        """
+        Returns the Cartan Matrix for the algebra. For a
+        given simple Lie algabra the elements `a_ij` can be
+        generated by
+
+        a_ji = 2 <\u03B1_i, \u03B1_j> / <\u03B1_j, \u03B1_j>
```

Comment:
Is it possible to use readable names like `\alpha` for the unicode characters?

Reply:
I can update them to that.